system-x/services/ldap/src/main/java/software/tnb/ldap/validation/LdapValidation.java

```diff
@@ -0,0 +1,44 @@
+package software.tnb.ldap.validation;
+
+import software.tnb.common.validation.Validation;
+
+import com.unboundid.ldap.sdk.LDAPConnection;
+import com.unboundid.ldap.sdk.LDAPException;
+import com.unboundid.ldap.sdk.SearchRequest;
+import com.unboundid.ldap.sdk.SearchResult;
+import com.unboundid.ldap.sdk.SearchResultEntry;
+import com.unboundid.ldap.sdk.SearchScope;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LdapValidation implements Validation {
+
+    private String ldapUsername;
+    private String ldapPassword;
+    private String ldapUrl;
+
+    public LdapValidation(String ldapUsername, String ldapPassword, String ldapUrl) {
+        this.ldapUsername = ldapUsername;
+        this.ldapPassword = ldapPassword;
+        this.ldapUrl = ldapUrl;
+    }
+
+    public List<String> getEntries(String objectClass) throws LDAPException {
+        LDAPConnection ldapConnection = new LDAPConnection("localhost", Integer.parseInt(ldapUrl.substring(ldapUrl.lastIndexOf(':') + 1)));
```

Comment:
this should be a substring between `ldap://` and `:` rather than hardcoded `localhost`, as the container might run on a different host when running locally (in windows testing)

Reply:
Done